frontend/pages/api/auth/[...nextauth].ts

```diff
@@ -5,13 +5,46 @@ import IAccount from "types/account";
 import iToken from "types/token";
 import IUser from "types/user";
 import ISession from "types/session";
+import axios from "axios";
```

Comment:
@NuggetOfOdium Can you use [fetch](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API) instead of axios?

Next.js already ships with a polyfill for fetch. So, it'd be good to stick with fetch.